specification/cognitiveservices/ContentSafety/models.tsp

```diff
@@ -489,6 +659,20 @@ enum ImageWithTextCategory {
   Violence,
 }
 
+@added(ContentSafety.Versions.v2024_01_30_Preview)
+@parentResource(TextCustomizedCategory)
+@resource("models")
+model TextTrainedModel {
+  @key
+  modelId: string;
+
+  status: ModelStatus;
+  createdTime: utcDateTime;
+  lastUpdateTime: utcDateTime;
+  warnings?: AsyncJobWarning[];
+  errors?: string[];
```

Comment:
Will need `@doc`s by GA.

Reply:
Will add.
Moved custom category to next version.